### PR TITLE
Chapter 11

### PR DESCRIPTION
**Pull request by 1hoan234**
Chapter 11

### CODE REVIEW

**Comment by 1hoan234** [comment]
ready

File: app/helpers/account_activations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (nhatnq-3267) [inline app/helpers/account_activations_helper.rb:1]: Xóa file thừa

File: app/models/user.rb [hunk 2]
```diff
@@ -45,11 +50,31 @@ def forget
...
+
   private
   def downcase_email
     email.downcase!
   end
+
+  # Creates and assigns the activation token and digest.
+  def create_activation_digest
+    self.activation_token = User.new_token
+    self.activation_digest = User.digest(activation_token)
```
Comment (nhatnq-3267) [inline app/models/user.rb:78]:
```suggestion
    activation_token = User.new_token
    activation_digest = User.digest(activation_token)
```

**Review by nhatnq-3267 — COMMENTED** [review]
Fix xong merged

**Pull request merged**